docs/components/clipboard/index.md

```diff
@@ -0,0 +1,181 @@
+---
```

Comment:
@kylebuch8 this file is stale, compared to the [current demo file](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-clipboard/demo/index.html). Are we attempting to keep these in sync, or is there a script that copies them?

Reply:
It's stale. We'll need to address keeping docs in sync in a later PR.